src/sentry/lang/native/applecrashreport.py

```diff
@@ -124,7 +124,7 @@ def _convert_frame_to_apple_string(self, frame, next=None, number=0):
         offset = ""
         if frame.get("image_addr") is not None and (
             not self.symbolicated
-            or (frame.get("function") or NATIVE_UNKNOWN_STRING) == NATIVE_UNKNOWN_STRING
```

Comment:
@wmak I recently saw these PRs, which may be thee reason why this particular style of value coercion was used.

https://github.com/getsentry/sentry/pull/15549
https://github.com/getsentry/sentry/pull/15602

/cc @untitaker

Reply:
yeah pretty sure that won't work

Reply:
Event normalization replaces invalid values with an explicit `None` in many cases. The default value in `get` is only applied if the key is missing completely, so it would be safer to leave the `or` here.

Reply:
Ah whoops sorry, didn't realize that. I'll flip these back :+1: